src/lib/types/api/apps.interfaces.ts

```diff
@@ -40,3 +40,13 @@ export interface App {
 
     dataProperties?: Record<string, unknown>
 }
+
+export interface PliancyApp {
```

Comment:
Our other types don't include a `Pliancy` prefix, so perhaps we can change to just using `App` for consistency

```suggestion
export interface App {
```

Reply:
@CalebAlbers , there was already an interface named App for the customer but just renamed that to CustomerApp and this to App:
https://github.com/pliancy/api-sdk/blob/9af1a737ab2d01983e9395f753ff4791ae1b558b/src/lib/types/api/apps.interfaces.ts#L26-L52

Reply:
That's a good call